Backend/Routes/AppRoutes/Project.route.js

```diff
@@ -1,47 +0,0 @@
-const {Router} = require('express');
-const {ProjectModel} = require('../../Models/Task.model');
-
-const ProjectController = Router();
-
-
-ProjectController.post('/', async(req,res)=>{
-    const {project_name, user_id} = req.body;
-    const project = new ProjectModel({
-        project_name,
-        user_id
-    })
-    await project.save();
-    return res.send({message : " new project added"})
-})
-ProjectController.get("/", async(req,res)=>{
-    const {user_id} = req.body;
```

Comment:
why is anyone allowed to see any project?
user can see the projects that are created by him or his team

i think same auth logic in backed can be added here as well